### DISCUSSION TITLE
docs/analytics

### DISCUSSION THREAD

**Discussion by giscus** [flowershow/flowershow#612] [Announcements]
# docs/analytics

Google Analytics (GA4) is currently supported.Set your google analytics key in the config.js file.
(See \[\[docs/config|this chapter]] about basic user configurations.)
...

https://flowershow.app/docs/analytics

**Comment by smmca** [comment]
https://plausible.io/ please!

**Reply by rufuspollock** [reply]
@smmca 👍 - see #238
